
Due to MODULEPATH changes, the following have been reloaded:
  1) libfabric/1.10.1     2) openmpi/4.0.3     3) ucx/1.8.0

RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd
Traceback (most recent call last):
  File "./main.py", line 2, in <module>
    import modules.util as util
  File "/scratch/endeavor/python_script_test/modules/util.py", line 2, in <module>
    import cv2
  File "/scratch/endeavor/python_script_test/tensorflow/lib/python3.8/site-packages/cv2/__init__.py", line 5, in <module>
    from .cv2 import *
ImportError: numpy.core.multiarray failed to import
